tests/Fixer/Phpdoc/PhpdocAlignFixerTest.php

```diff
@@ -1454,4 +1454,44 @@ public function testTypesContainingWhitespace(): void
              */
         ');
     }
+
+    public function testTypesContainingBackslash(): void
+    {
+        $this->doTest('<?php
+            /**
+             * @var string                            $input
+             * @var \Closure                          $fn
+             * @var \Closure(bool):int                $fn2
+             * @var Closure                           $fn3
+             * @var Closure(string):string            $fn4
+             * @var array<string,array<string,mixed>> $data
+             */
+            /**
+             * @param string                            $input
+             * @param \Closure                          $fn
+             * @param \Closure(bool):int                $fn2
+             * @param Closure                           $fn3
+             * @param Closure(string):string            $fn4
+             * @param array<string,array<string,mixed>> $data
+             */
+            /**
+             * @var string                   $value
+             * @var \Closure(string): string $callback
+             * @var Closure(int): bool       $callback2
+             */
+            /**
+             * @param string                   $value
+             * @param \Closure(string): string $callback
+             * @param Closure(int): bool       $callback2
+             */
+            /**
+             * @var Closure(array<int,bool>): bool $callback1
+             * @var \Closure(string): string       $callback2
+             */
+            /**
+             * @param Closure(array<int,bool>): bool $callback1
+             * @param \Closure(string): string       $callback2
```

Comment:
could we have not only `clean` version that is looking good, but also ugly version before fixing, to make sure that we not only don't break nice looking code, but that we also fix ugly code?

Reply:
I've added some ugly versions.